docs/practical_information/README.md

```diff
@@ -51,6 +55,8 @@ and will be downloaded automatically from Docker Hub when you run the `docker` o
 
 #### Using Docker
 
+Please note that docker requires root access, which is available to you in the tutorials AWS environment.
```

Comment:
No, `docker` will be pre-installed in the AWS environment, but you don't need root in the AWS env at all (the Docker daemon runs as root, but that's irrelevant to the tutorial participants).

Reply:
Context brainfart. Thanks. Removed.